setup.py

```diff
@@ -63,14 +62,10 @@ def get_install_requires():
         "paasta_tools/kubernetes/bin/kubernetes_remove_evicted_pods.py",
         "paasta_tools/kubernetes/bin/paasta_cleanup_stale_nodes.py",
         "paasta_tools/kubernetes/bin/paasta_secrets_sync.py",
-        "paasta_tools/list_marathon_service_instances.py",
         "paasta_tools/log_task_lifecycle_events.py",
-        "paasta_tools/marathon_dashboard.py",
         "paasta_tools/monitoring/check_capacity.py",
-        "paasta_tools/monitoring/check_marathon_has_apps.py",
         "paasta_tools/monitoring/check_mesos_active_frameworks.py",
         "paasta_tools/monitoring/check_mesos_duplicate_frameworks.py",
-        "paasta_tools/monitoring/check_mesos_outdated_tasks.py",
         "paasta_tools/monitoring/check_mesos_quorum.py",
```

Comment:
probably also deleteable :)

Reply:
Yeah I've got other PRs up for deleting a bunch of things we noticed while working on the merge conflicts for this one.

Didn't want to make this PR any bigger than it needed to be.